src/bindings/js/node/tests/core.test.js

```diff
@@ -43,11 +43,25 @@ it('Core.setProperty(\'CPU\', { \'NUM_STREAMS\': 5 })', () => {
   assert.equal(result, streams);
 });
 
-it('Core.setProperty(\'CPU\', { \'INFERENCE_NUM_THREADS\': 3 })', () => {   
```

Comment:
Let's fix old code-style issues in a separate PR dedicated to eslint fixes

Reply:
My IDE did this automatically (removed trail spaces), do you want to rollback these changes?